### ISSUE TITLE
Sleeker UI

### ISSUE THREAD

**Issue by Hulkhulkthehulk** [Huzuni/Huzuni-1.11#10]
The old 1.8 UI was so perfect: everything could be hidden, mostly keybind reliant rather than mouse (moving around the new menu is incredibly time consuming whereas the old menu displayed all mods at once and allowed easy access to settings)
this was my favorite client for a really long time because of the "hide everything" aspect of it that saved my ass a dozen times during screenshares, and it seems like you guys took it in a totally wrong direction from where it was

**Comment by brudin** [comment]
You can still hide everything in-game, if you open the GUI and look at the 'Widgets' tab you can enable/disable/configure individual HUD elements and if you click the 'Settings' button to hide the menu you can drag them around.
We're going to re-add the old 'Hide huzuni' feature in the next update, and we also have a new GUI in the works that will be much easier to use, here's a picture of what it looks like right now: http://i.imgur.com/dmCygeM.png
We are also considering adding something like the old GUI back as an option since a lot of people like that.

**Comment by Hulkhulkthehulk** [comment]
That looks pretty awesome
might I ask why the site was down for so long, and why you chose to move to forge?

**Comment by brudin** [comment]
It was down because the host we used for the site couldn't handle all of the traffic, no joke!  We're using 

Github's web hosting thing now which is much more reliable.
We have always wanted to make the client work with Forge, but only recently figured out a way of doing it without having to make any compromises.

On the topic of updating old versions, this is where the fun part comes in!  So the main issue right now is that Minecraft's code changes a lot between updates, which makes maintaining older versions difficult... though I think I have come up with a solution for this, and it's something I've been working on for a week or so now.

Basically we're removing all direct calls to Minecraft code from Huzuni, and instead pointing it to our own 'wrapper' classes.  This means that we will only have to update the wrapper classes when it comes to updating the client, and will be able to release the same Huzuni for multiple versions effortlessly.

It's a pretty long process for us to port all of our code over to this method, but will definitely be worth it.  We'll be able to release the client for 1.8-1.11.2, and might release a Liteloader version for people who don't like Forge.

**Comment by Hulkhulkthehulk** [comment]
Props for the dedication, I and many others honestly thought this client was dead for a really long time.

Can't wait for updates!

**Comment by brudin** [comment]
No worries, I love bug reports. These are for the 1.8 version, correct?  

I'm pretty sure I know what's causing the chat bug, I think for now we're going to remove the custom chat option and replace it with something using the new GUI api thing later.  

The issue with OptiFine not showing on the main menu is because we set the main menu to use our own main menu class, while OptiFine has the text show up on the vanilla one.  This could be solved a number of ways and I'll look into seeing which way would be the best.  This and the chat bug will both be fixed when we 'unify' the different versions, which will hopefully be soon.

Kill Aura and ESP will only show up on the enabled mods widget if you have a keybind set (which you do) and you have 'displayable' enabled,  http://i.imgur.com/gjnY9e7.png

If you find anymore bugs let me know!

**Comment by Hulkhulkthehulk** [comment]
Yep, all 1.8.8 with the latest version of forge for 1.8.8

Realized I had "displayable" disabled for them, which is strange since I definitely hadn't touched the ESP settings other than setting my keybind. Maybe some aren't enabled by default?

**Comment by brudin** [comment]
With the 'old school' theme, it'll only render 'displayable' mods with keybinds bound, that's intentional. :)

Nametags are definitely something that need to be reworked... that issue with overlapping has been bothering me for a while too.

Xray's issues with transparent blocks is fixed for the next release (I hope!).  If you're talking about water/lava, make sure you have them enabled.

I definitely want to have the search menu again for the Xray - Xray config in general hasn't been implemented in the new GUI we're working on but is definitely one of the top priorities for it, I use that a lot and miss being able to search through all of the blocks.

**Comment by Hulkhulkthehulk** [comment]
Along with nametags, the "display armor" thing doesn't show the number of items in the hand anymore, only the item itself. 

I didn't mean water and lava, I meant the weird textures showing up. glad that'll be fixed haha

**Comment by brudin** [comment]
Yeah, it doesn't sprint in water since there's no speed gain from sprinting in water, last I checked.  Just fixed the flying issue, though.

**Comment by Hulkhulkthehulk** [comment]
bit of a niche case, but depth strider 3 sprinting is faster than depth strider 3 walking
https://www.youtube.com/watch?v=CZF88kiCMq8 proof at like 1:58 if you don't believe me

**Comment by brudin** [comment]
Huh... interesting.  I'll look into this, thanks!

**Comment by Hulkhulkthehulk** [comment]
https://vid.me/wGlA
The bow aimbot seems to have some glitches, notably that it makes your head snap up before every shot
also does this against players
showed my reach setting at the end so you know it's not pointing at another mob in the distance

**Comment by Dracers** [comment]
Uhhm, I know its not a bug or something... But when I get into huzuni into forge, I'm  just dealing a lots of frame lags or frame drops[FPS]... Can I ask what's the problem of this? :l 
And, I insert Optifine 1.11.2 to mods, after that I can't load server textures and it straight timed out me. I hope you'll fix this asap thanks! >w<

**Comment by brudin** [comment]
@Hulkhulkthehulk That's intentional behavior due to arrow gravity, I think.

@Dracers Not sure what would be causing frame drops, though the next update will have some decent performance boosts.  In regards to optifine issues, did it crash?  If so, can you provide a crash log?

**Comment by Dracers** [comment]
@brudin For optifine [forge + huzuni + optifine], it just timed out me, but without optifine [huzuni + forge], it works in high chance but it hangs for some seconds like 20+.. It doesn't crash or anything, the server textures is just as simple as that. I just get a timed out after its done loading.. [Hypixel - Blocking Dead Texture Pack]
